config/default/user.role.intranet_viewer.yml

```diff
@@ -1,9 +1,10 @@
 uuid: c3d0e7ba-c576-4abb-a3e1-5d2db98c0892
 langcode: en
 status: true
-dependencies: {  }
+dependencies: null
 id: intranet_viewer
 label: 'Intranet Viewer'
 weight: -6
 is_admin: null
-permissions: {  }
+permissions:
```

Comment:
No permissions should be for this role. it is meant to be just an identifier.

Reply:
Removed the permission, it was not being used because that functionality was reimplemented in another way.